docs/configuration/system/conntrack.rst

```diff
@@ -176,22 +176,27 @@ Conntrack ignore rules
 Conntrack log
 =============
 
-.. cfgcmd:: set system conntrack log icmp destroy
-.. cfgcmd:: set system conntrack log icmp new
-.. cfgcmd:: set system conntrack log icmp update
-.. cfgcmd:: set system conntrack log other destroy
-.. cfgcmd:: set system conntrack log other new
-.. cfgcmd:: set system conntrack log other update
-.. cfgcmd:: set system conntrack log tcp destroy
-.. cfgcmd:: set system conntrack log tcp new
-.. cfgcmd:: set system conntrack log tcp update close-wait
-.. cfgcmd:: set system conntrack log tcp update established
-.. cfgcmd:: set system conntrack log tcp update fin-wait
-.. cfgcmd:: set system conntrack log tcp update last-ack
-.. cfgcmd:: set system conntrack log tcp update syn-received
-.. cfgcmd:: set system conntrack log tcp update time-wait
-.. cfgcmd:: set system conntrack log udp destroy
-.. cfgcmd:: set system conntrack log udp new
-.. cfgcmd:: set system conntrack log udp update
-
-    Log the connection tracking events per protocol.
\ No newline at end of file
+    Log the connection tracking events per type.
```

Comment:
please switch all the text and the command it belongs to. So the explanation have to be under the command.

Reply:
fixed